README.md

```diff
@@ -197,10 +197,17 @@ yarn run pilet publish --no-fresh --url "https://frontends.service.eu.axinom.net
 ```
 
 Please note, that you need to provide the 'Base64 Encoded Credentials' of a
-service account that has the `Pilets: Publish` permissions as `api-key`. This
+[service account](https://docs.axinom.com/platform/core/identity/authenticate-serviceaccount/#what-is-a-service-account) 
+that has the `Pilets: Publish` permissions under the "Micro Frontend Service" as `api-key`. This
 value will be shown to you alongside with the other secrets when creating a
 service account or when generating new secrets for an existing account.
 
+For authentication, you need a Client ID and Client Secret of this service account. 
```

Comment:
this should either be removed, or moved further up.
As this is the general info, that the call needs the service account details - and then we can explain which permissions it needs and how to obtain it:
1. on creation we show the calculate value
2. if you have the plaintext values already, you can use the base64 tool, etc

Reply:
Fixed